src/rsa.cpp

```diff
@@ -106,6 +106,29 @@ sp_rsa_key rsa::load_from_file(const std::string &path, password_cb on_password)
 	return std::shared_ptr<RSA>(r, ::RSA_free);
 }
 
+struct BIODeleter {
```

Comment:
Mind changing it to `class` and move declaration to `jwtpp.hh` ?

Reply:
Will do